hubblestack/utils/stdrec.py

```diff
@@ -45,6 +45,8 @@ def std_info():
         'dest_ip': fqdn_ip4,
         'dest_fqdn': local_fqdn,
         'system_uuid': __grains__.get('system_uuid'),
+        'timezone': __grains__.get('timezone'),
```

Comment:
Can we remove the these new fields so they won't be sent in with _every_ event. Since these are in grains they will be sent in at configurable intervals that are normally adequately frequent.

In addition, there is a naming collision with an existing timezone grain. The existing timezone grain for some reason is not populated every time, but it does exist occasionally.

Reply:
Ah, good point. We don't need to put the new grains anywhere in the splunk returners since grains are already reported hourly.

Let's call the timezone `timezone_short` since it's the abbreviation anyway.